docker/backend/Dockerfile

```diff
@@ -121,7 +121,7 @@ RUN set -x \
 RUN ./scripts/docker/install-workers-deps.sh
 
 COPY --from=golang "/root/scc" "/root/scc"
-COPY --from=golang "/root/scorecard/scorecard" "./scorecard"
+COPY --from=golang "/root/scorecard/scorecard" "/root/scorecard"
```

Comment:
@MoralCode : I've ignored this because its in DRAFT mode. Should we do it?

Reply:
oh, uuh if its been causing issues then sure - or maybe at least for consistency?

@cdolfi have you seen errors relating to this?